CHANGES.md

```diff
@@ -1,3 +1,16 @@
+## [Pending next release]
+
+RPC changes:
+- Tendermint-Go v0.33 compatibility - issue #184
+  - `abci_info`, `abci_query`, `block_results`, `genesis` structs
+  - serialization/deserialization fixes
+  - Updated/fixed integration tests
+
+CI changes: - issue #120
+- Moved to GitHub Actions
+- Updated crates.io badges
+- Enabled integration tests in CI with Tendermint-Go node service
```

Comment:
Not sure how relevant CI changes are to readers of the changelog because these do not have any effect on users of tendermint-rs. On the other hand it's probably good to communicate what happened since the last release anyways. Note that the other badges were also updated in a another PR/issue.